node/test/scripts/print-output-exe.cs

```diff
@@ -0,0 +1,39 @@
+using System;
```

Comment:
this is a small program used on windows only for reliably testing the tool runner feature for piping the output from one program to another

this is the first program in the chain